libs/signed_integers/src/i128.sw

```diff
@@ -206,3 +207,17 @@ impl core::ops::Subtract for I128 {
         res
     }
 }
+
+// impl TwosComplement for I128 {
```

Comment:
This is commented out, was this done on purpose?

Reply:
I'm guessing this depends on implementing `NOT` for `I128`, and same for `I256`?

Reply:
Yes, as well as bumping the dependencies in the repo